lifecycle-lab/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './App.css';
+import Todo from './Component/Todo';
+import {toast} from 'react-toastify';
+import Form from './Component/Form'
+import 'react-toastify/dist/ReactToastify.css';
+class App extends React.Component {
+
+  state = {
+    toDoList:[]
+  }
+  addToDo =(toDo)=>{  
+    this.setState({toDoList:[...this.state.toDoList , toDo]})
```

Comment:
it would be better to use the callback method to update the current state since this sometimes doesn't work as expected